docs/payid-headers.md

```diff
@@ -27,7 +27,9 @@ Some different header options are shown here, with example values.
 | BTC      | `application/btc-mainnet+json`  | { address: '1BvBMSEYstWetAu4m4GFg7xJaNVN2' }                      |
 | XRP      | `application/xrpl-mainnet+json` | { address: "rw2ciyaNshpHe7bCHo4bRWq6pqqynnWKQg", tag: "67298042"} |
 | ACH      | `application/ach+json`          | { account: '363023456079',routing: '011302838'}                   |
-| All      | `application/payid+json`        | Variable depending on the contents of each address                |
+| All      | `application/payid+json`        | Variable, depending on the contents of each address               |
+
+If you create your own Accept header for another currency, follow the pattern for BTC Accept headers, and use three characters for the currency, as shown.
```

Comment:
We should also ask them to make a PR to register it here?

@sappenin , is the docs website the right place to "register" a currency, or do we have/need a IANA-like list of networks and associated headers?

Reply:
@hbergren Let's put this in the Docs site for now. Once it grows and becomes more widely-deployed, we can look to make an RFC, but for now this is a good place to located this.

Reply:
There already exists a IANA registration section in the PayID protocol RFC for Media-type. See section 14.2 of https://github.com/payid-org/rfcs/blob/master/dist/spec/payid-protocol.txt

Reply:
> There already exists a IANA registration section in the PayID protocol RFC

Ah yes, I overlooked that. I think this should still live on the docs page, so maybe we adjust the note around "new types"?

@aanchal4 did you have a particular suggestion here?

Reply:
FWIW, I opened https://github.com/payid-org/payid-documentation/pull/99 to capture @aanchal4's comment.